code/game/objects/machinery/miner.dm

```diff
@@ -216,7 +224,9 @@
 	if(miner_status != MINER_RUNNING)
 		to_chat(user, "<span class='warning'>[src] is damaged!</span>")
 		return
-
+	if(miner_upgrade_type == "mining computer")
```

Comment:
Use defines for this

Reply:
The define is always returning true here what do